db/schema.rb

```diff
@@ -58,4 +58,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "email"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "password_digest"
+    t.index ["email"], name: "index_users_on_email", unique: true
+  end
+
```

Comment:
empty line here

Reply:
this is auto-generated file, so maybe I should leave this like that?